src/meshapi/urls.py

```diff
@@ -39,6 +39,7 @@
         views.DisambiguateInstallOrNetworkNumber.as_view(),
         name="meshapi-v1-disambiguate-number",
     ),
+    path("uisp-import/nn/<network_number>/", views.uisp_import_for_nn, name="uisp-import-by-nn"),
```

Comment:
Maybe there should be a type here: `<int:network_number>`? I think that makes it 404 automatically if they give us letters

Reply:
```suggestion
    path("uisp-import/nn/<int:network_number>/", views.uisp_import_for_nn, name="uisp-import-by-nn"),
```